### ISSUE TITLE
when mistakes are made, excessive backspace is often required

### ISSUE THREAD

**Issue by evgenyfadeev** [didoesdigital/typey-type#191]
That is - when I practice and enter a word incorrectly I often find needing to type backspace way more than the number of strokes necessary to erase the word I've mistyped.

Sometimes the input that appears even seemingly from a previous exercise.

Is that an issue with the react state in this app or a feature of plover?

**Comment by didoesdigital** [comment]
@evgenyfadeev I'm not sure I understand exactly what you're seeing. Can you please share an example of this? Perhaps a description of specific words or a video. And can you confirm if you mean you've typed an actual "backspace" or if you mean the steno "undo stroke" functionality (which is usually the `*` star key)?

My best guess is that you've pressed keys that would result in a multi-stroke word. For example, if you had the material "sigh" followed by "lent" and wrote `SAOEU` then misstroked `HREPBT` instead of `HREPBD`, Plover would see that as `SAOEU/HREPBT` and try to back track to the previous word to re-write it like changing "sigh" to "silent". Typey Type would have already marked the first word "sigh" as correct and deleted it so you would only get "lent". Then pressing `*` to undo the `HREPBT` stroke would leave you with some weird text like "gh" from the end of "sigh".

**Comment by evgenyfadeev** [comment]
@didoesdigital yes - when I type multistroke words and make a mistake, after a while typing * repeatedly would bring out more and more of the previously typed input. I see sometimes the same behaviour in the Plovel lookup tool as well, that's why I suspect it might be a feature of the Plover software.

I'll try to record a screencast and share it somehow.

Thanks.

**Comment by evgenyfadeev** [comment]
@didoesdigital here is a short snipppet. I've typed word "misery", the word was accepted then and proceeded to "swampiest", made some mistakes and then when typing the * "misery" popped back in again.

https://jam.dev/c/1bc5ac54-8f61-4669-bcf8-0002a99eb2ac

I'll practice a bit more and see if I can make a more obvious example.

**Comment by didoesdigital** [comment]
@evgenyfadeev if you did a custom lesson with this material in Typey Type:

```
fish	TPEURB
cat	KAT
```

… would you typically write only `TPEURB KAT` or do you usually have to write `TPEURB * KAT`?

**Comment by evgenyfadeev** [comment]
@didoesdigital no problems with the "fish cat" exercise on my side.

**Comment by evgenyfadeev** [comment]
@didoesdigital no problems with "TPEURB * KAT"
